user pojo changed

ManasviniArul opened the pull request:
added pojo class and made changes.

Commit:
user pojo changed

grewaviv reviewed (approved):
These are the columns of user table which is a persistent storage. A fetch on user table will display the fields in the user table. If you store age it will be remain the same always if you don't update it. Age can't be fixed it changes every year on the other hand dob is fixed. 

If you want to display age you can always calculate it from dob

ManasviniArul commented:
> These are the columns of user table which is a persistent storage. A fetch on user table will display the fields in the user table. If you store age it will be remain the same always if you don't update it. Age can't be fixed it changes every year on the other hand dob is fixed.
> 
> If you want to display age you can always calculate it from dob

I removed age and added dob. is it okay can I merge now
